server/views/whats-new.njk

```diff
@@ -20,6 +20,25 @@
             <h1 class="govuk-heading-l">What's new</h1>
             <p class="govuk-body">The latest updates and improvements to Prepare a case for sentence.</p>
 
+            <h2 class="govuk-heading-m">25 August 2023</h2>
+
+            <h3 class="govuk-heading-s">New filters on the case list</h3>
+
+            <p class="govuk-body">You can now filter the case list by</p>
+
+            <ul class="govuk-list govuk-list--bullet">
+                <li>source - so you can just see Common Platform cases or just Libra cases</li>
+                <li>flag - so you can see only breach cases</li>
+            </ul>
+
+            <p class="govuk-body">We hope this will help you find the cases you need more quickly.</p>
+
+            <h3 class="govuk-heading-s">Speed improvements</h3>
+
+            <p class="govuk-body">We have made some changes in the background of the service so that pages will load more quickly. If you have seen an error that says 'There is a problem with the service, try again later' recently, one of the reasons why is because the pages are loading too slowly. </p>
```

Comment:
Should we be specific that the improvements are specific to case list page?

Reply:
I think that's a discussion for the ticket and/or stand up? But agree you are right there with the terminology

Reply:
Updated from slack content